SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
 # Цвет координат курсора мыши на Карте Приключений
 # Die Koordinaten des Mauszeigers während man auf der Abenteuerkarte ist
 # Συντεταγμένες του κέρσορα του ποντικιού όταν βρίσκεστε στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoord = H3Cyan
+AdventureCoord = Regular
 
 # The color of hit points of the creature under the mouse cursor while in combat
 # Цвет здоровья стека существ при наведении курсора мыши во время боя
 # Die Farbe der Lebenspunkte der Kreatur unter dem Mauszeiger während der Kampf
 # Το χρώμα των πόντων ζωής του πλάσματος κάτω από τον κέρσορα του ποντικιού κατά τη διάρκεια της μάχης
-HitPoints = H3Orange
+HitPoints = Regular
 
 # The color used when all targetted creatures will perish / or be resurrected
 # Цвет, означающий, что все целевого стека существ погибнут или будут воскрешены
 # Die benutzte Farbe wann alle gezielten Kreaturen sterben werden / oder sie auferstanden werden werden
 # Το χρώμα που χρησιμοποιείται όταν όλα τα στοχευμένα πλάσματα θα πεθάνουν / ή θα αναστηθούν# Το χρώμα που χρησιμοποιείται όταν μόνο ένα τμήμα των στοχευμένων πλασμάτων θα πεθάνει / ή θα αναστηθεί
-KillAll = H3Green
+KillAll = Regular
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.